docs/Assets/AddingAnimationsFromUnity.md

```diff
@@ -3,12 +3,76 @@ id: AddingAnimationsFromUnity
+
+Start by creating separate Sub-State Machines for all of the intended visuals. In this guide we'll start off by creating three simple visuals: Idle, Eat, and Run. A generic "Control" state will be placed first to allow for additional logic later if needed.
+
+![Sub-state machine example](/img/product/animation/anim_import_8.png)
+
+To control when these different flows are started, we'll add parameters. Parameters can be of type Float, Int, Bool, or Trigger. Triggers are like Bools, but will automatically switch off after being used. In this setup we'll use a Bool for Run so that the wolf will always be running until explicitly told to stop. A Trigger will be used for Eat, so it can be set once and not have to be worried about after. Parameter names can be called anything, but it's helpful to name them after the animations that you'll use them to drive.
+
+![A state flow](/img/product/animation/anim_import_9.png)
+
+Next, Transitions need to be setup from the Control state to each of our flows. Transitions are made by right clicking a state, selecting Make Transition, then clicking the destination for the transition.
+
+![Make a transition](/img/product/animation/anim_import_10.png)
+
+Clicking an individual Transition line lets you see the properties associated with that Transition. For each flow, you'll have to add entries to the Conditions section to link the parameters to a Transition.
+
+![Make a transition](/img/product/animation/anim_import_11.png)
+
+> Transitions are evaluated in the order they are created. The order can be modified by selecting the source node and dragging its Transitions in the Inspector.
+
+![Transition inspector](/img/product/animation/anim_import_12.png)
+
+Double clicking a Sub-State Machine will enter it, and allow individual states to be added. Once a state has been created, you'll have to assign a Motion from the Inspector. The motions will have the same names as the imported animations from the first part of thie guide. It's important to setup a Transition to the Exit node, so that the flow will leave the Sub-State Machine. Depending on your desired effect, you may want to have multiple Transitions to exit. This can be useful for allowing immediate changes to other flows, or for forcing a flow to finish before it exits, regardless of what happens with the parameters.
+
+![Inside the sub-state machine](/img/product/animation/anim_import_13.png)
+![Sub-state machine flows](/img/product/animation/anim_import_14.png)
+
+More complex flows can be created inside of a Sub-State Machine. For instance, if the wolf is always supposed to howl before sprinting, all of that can be controlled in the Run flow. In this setup, there's a Howl state and a Run state. The Run state has a two Transitions on it. One leads to Exit if the Run parameter switches to false. The other transitions back to itself, so the wolf will continuously run until stopped.
+
+![Sub-state machine flows](/img/product/animation/run_anim.gif)
```

Comment:
This filename uses a hyphen instead of an underscore.  Need to either change the filename to match the usage here or change the usage here to match the filename.

Reply:
wow, can't believe I missed that.